management-app/src/main/java/com/sdevelopment/skt/management/controller/ProductController.java

```diff
@@ -0,0 +1,63 @@
+package com.sdevelopment.skt.management.controller;
+
+import com.sdevelopment.skt.common.domain.Product;
+import com.sdevelopment.skt.management.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import javax.validation.Valid;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+@Controller
+public class ProductController {
+
+    @Autowired
+    private ProductService productService;
```

Comment:
Is it better to use field injection rather than constructor injection?

Reply:
After research it's better constructor injection.

Reply:
Changed to constructor injection.